Assets/MixedRealityToolkit.SDK/Inspectors/Utilities/Solvers/TapToPlaceInspector.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@Troy-Ferrell is in the middle of adding a guideline about this, but I don't think this component needs a custom inspector. The guideline is "avoid writing custom inspectors unless needed".

I don't see why this component should be frozen / not modifiable when in play mode, and the collider warning is a nice to have, not must have. The benefit of not needing to maintain 2 files seems to outweigh this small benefit.

Reply:
Addressed [here](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7188#discussion_r381608858)